src/pk_compiler.c

```diff
@@ -190,7 +190,7 @@ static _Keyword _keywords[] = {
   { "while",    5, TK_WHILE    },
   { "for",      3, TK_FOR      },
   { "if",       2, TK_IF       },
-  { "elif",     4, TK_ELIF     },
+  { "else if",  4, TK_ELSEIF   },
```

Comment:
it should be `"elsif"` and the 4 next to is the length of the keyword, which should be updated to 5 after the rename (and note that a keyword cannot contain a white space).

Reply:
Changed the keyword count as well.